### PR TITLE
[Merged by Bors] - chore: move style linters to `Style`

### PR DESCRIPTION
**Pull request by grunweg**
This makes for a more coherent split of the linter files. Third attempt after #15942 and #16510.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Tactic/Linter/Lint.lean [hunk 1]
```diff
@@ -96,335 +97,3 @@ def dupNamespace : Linter where run := withSetOptionIn fun stx ↦ do
 initialize addLinter dupNamespace
 
 end DupNamespaceLinter
```
Comment (adomani) [inline Mathlib/Tactic/Linter/Lint.lean:99]: Can you make sure that there are no "open" namespaces here?

**Review by adomani — COMMENTED** [review]
Thanks!

Just a minor check

bors d+

File: Mathlib/Tactic/Linter/Lint.lean (hunk 1 above)
Comment (grunweg) [reply Mathlib/Tactic/Linter/Lint.lean]: Good catch, fixed!

**Comment by grunweg** [comment]
Thanks for the fast review!
bors merge

**Pull request closed without merging**